cypress/commons/actions/generic/DatasetManager.js

```diff
@@ -15,8 +15,81 @@ function switchToDatasetManagerView() {
   return getDatasetManagerTab().click();
 }
 
+function getNoDatasetsPlaceholder() {
+  return cy.get(GENERIC_SELECTORS.datasetmanager.noDatasetsPlaceholder);
+}
+
+function getCreateDatasetButton() {
+  return cy.get(GENERIC_SELECTORS.datasetmanager.createDatasetButton);
+}
+
+function getDatasetMetadataCard() {
+  return cy.get(GENERIC_SELECTORS.datasetmanager.metadata.card);
+}
+
+function getDatasetMetadataAuthor() {
+  return cy.get(GENERIC_SELECTORS.datasetmanager.metadata.author);
+}
+function getDatasetMetadataCreationDate() {
+  return cy.get(GENERIC_SELECTORS.datasetmanager.metadata.creationDate);
+}
+function getDatasetMetadataRefreshDate() {
+  return cy.get(GENERIC_SELECTORS.datasetmanager.metadata.refreshDate);
+}
+function getDatasetMetadataSourceType() {
+  return cy.get(GENERIC_SELECTORS.datasetmanager.metadata.sourceType);
+}
+function getDatasetMetadataApiUrl() {
+  return cy.get(GENERIC_SELECTORS.datasetmanager.metadata.apiUrl);
+}
+function getDatasetMetadataApiUrlButton() {
+  return cy.get(GENERIC_SELECTORS.datasetmanager.metadata.apiUrlButton);
+}
+function getDatasetMetadataEditDescriptionButton() {
```

Comment:
Is there really an edit description button ?

Reply:
Yes, it's the pencil icon that appears when hovering the dataset description